### DISCUSSION TITLE
Support for OpenAI.client.chat.completions.with_raw_response

### DISCUSSION THREAD

**Discussion by aman-gupta-doc** [langfuse/langfuse#3062] [Ideas]
### Describe the feature or potential improvement

Hi,
Please provide support of `with_raw_response` of OpenAI SDK in langfuse SDK. So that we can drop-in replace OpenAI SDK with langfuse.

Also discussed in https://github.com/langfuse/langfuse/issues/2218

**Comment by marcklingen** [comment]
Thank for adding this, open for contributions in case someone wants to add this to the openai integration for python/js!

Added issue here: https://github.com/langfuse/langfuse/issues/3064

**Comment by Tailen** [comment]
I'm happy to work on python sdk if this is not added yet.
